doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='process-instance-migration-installing-service-proc']
+= Installing the process instance migration service
```

Comment:
We need to change this chapter. The procedure below applies to building from community bits, that's not what users should do (if they don't insist). I'll provide the steps shortly.

Reply:
1. Download rhpam-7.4.0-add-ons.zip from CSP.
2. Unzip following entry - rhpam-7.4-process-migration-service-standalone.jar to arbitrary location
3. The service requires kieserver configuration, provided in a yaml file (on top of thorntail specific config):
```config.yml
kieservers:
  - host: http://localhost:8080/kie-server/services/rest/server
    username: kieserver
    password: kieserver123*
  - host: http://localhost:8280/kie-server/services/rest/server
    username: kieserver
    password: kieserver1!
```
example contents of such file can then be:
```config.yml
thorntail:
  deployment:
    process-migration.war:
      jaxrs:
        application-path: /rest
      web:
        login-config:
          auth-method: BASIC
          security-domain: pim
        security-constraints:
          - url-pattern: /*
            roles: [ admin ]
          - url-pattern: /health/*
  datasources:
    data-sources:
      pimDS:
        driver-name: h2
        connection-url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
        user-name: sa
        password: sa
  security:
    security-domains:
      pim:
        classic-authentication:
          login-modules:
            UsersRoles:
              code: UsersRoles
              flag: required
              module-options:
                usersProperties: application-users.properties
                rolesProperties: application-roles.properties
kieservers:
  - host: http://localhost:8080/kie-server/services/rest/server
    username: kieserver
    password: kieserver123*
  - host: http://localhost:8280/kie-server/services/rest/server
    username: kieserver
    password: kieserver1!
```
4. startup the service by:
```
java -jar rhpam-7.4-process-migration-service-standalone.jar -s./config.yml
```
or when need to run at a port offset, add following option `-Dswarm.network.socket-binding-groups.standard-sockets.port-offset=200`

5. By default service is started on localhost:8080, may vary based on your port offset

Reply:
please do change the ports across the docs, 8080 is default when running like this, not 3300 as currently mentioned.

Reply:
@jstastny-cz Is the "-process-migration-service-standalone" file ".jar" or ".zip"?

Reply:
Done.